ProberI/CP1-amity-allocation
# PR1-Amity-code-review-PR

#### What does this PR do?
  * Amity room allocation CLI implementation.
  * OOP implementation.
  * Docopt for CLI 'front-end'.
#### Description of Task to be completed:
       Have 70% tests coverage.
       Have functions implemented according to python best practices.
#### How should this be manually tested?
   * Clone the repo.
   * Install project requirements in a virtual environment.
   * CD into app directory.
   * Run nose tests --with-coverage to check for test coverage.
   * Run python view.py -i to start app in interactive mode.
